runtime/help/commands.md

```diff
@@ -107,14 +107,19 @@ quotes here but these are not necessary when entering the command in micro.
 
 * `plugin available`: show available plugins that can be installed.
 
-* `reload`: reloads all runtime files.
+* `reload`: reloads all runtime files (settings, keybindings, syntax files,
+   colorschemes, plugins). All plugins will be unloaded by running their
+   `deinit()` function (if it exists), and then loaded again by calling the
+   `preinit()`, `init()` and `postinit()` functions (if they exist).
 
 * `cd 'path'`: Change the working directory to the given `path`.
 
 * `pwd`: Print the current working directory.
 
 * `open 'filename'`: Open a file in the current buffer.
 
+* `reopen`: Reopens the current buffer from disk.
```

Comment:
nit: "Reopens the current file from disk" would be more accurate?

Reply:
Yep, both this and your other nit make sense. I was actually going to write it the way you did, but decided to be conservative and just slightly adapt the doc comment for ReopenCmd:

```go
// ReopenCmd reopens the buffer (reload from disk)
func (h *BufPane) ReopenCmd(args []string)
```

Anyway, I went ahead and changed both to the nicer version.